python-checks/src/main/java/org/sonar/python/checks/PytzTimeZoneInDatetimeConstructorCheck.java

```diff
@@ -19,64 +19,65 @@
  */
 package org.sonar.python.checks;
 
-import java.util.List;
-import java.util.Objects;
 import org.sonar.check.Rule;
 import org.sonar.plugins.python.api.PythonSubscriptionCheck;
 import org.sonar.plugins.python.api.SubscriptionContext;
 import org.sonar.plugins.python.api.symbols.Symbol;
-import org.sonar.plugins.python.api.tree.ArgList;
-import org.sonar.plugins.python.api.tree.Argument;
 import org.sonar.plugins.python.api.tree.CallExpression;
 import org.sonar.plugins.python.api.tree.Name;
 import org.sonar.plugins.python.api.tree.RegularArgument;
 import org.sonar.plugins.python.api.tree.Tree;
 import org.sonar.python.cfg.fixpoint.ReachingDefinitionsAnalysis;
+import org.sonar.python.tree.TreeUtils;
 
 @Rule(key = "S6887")
-public class DateTimeNoPyTzTimeZoneInConstructor extends PythonSubscriptionCheck {
+public class PytzTimeZoneInDatetimeConstructorCheck extends PythonSubscriptionCheck {
 
   private ReachingDefinitionsAnalysis reachingDefinitionsAnalysis;
-  private static final String MESSAGE = "pytz.timezone should not be passed to the datetime.datetime constructor";
+  private static final String MESSAGE = "Don't pass a \"pytz.timezone\" to the \"datetime.datetime\" constructor.";
+  private static final String SECONDARY_MESSAGE = "The pytz.timezone is created here";
```

Comment:
I'm not sure where we can find guidelines for secondary messages and I don't even remember what they are, but it seems we have a tendency to have dots at the end of them (it is quick fix messages that don't have them).